CHANGELOG.md

```diff
@@ -34,6 +34,7 @@
 - Enh #862: Refactor PHP type of `ColumnSchemaInterface` instances (@Tigrov)
 - Enh #865: Raise minimum PHP version to `^8.1` with minor refactoring (@Tigrov, @vjik)
 - Enh #798: Allow `QueryInterface::one()` and `QueryInterface::all()` to return objects (@darkdef, @Tigrov)
+- Chg #872: Add `#[\SensitiveParameter]` attribute for password (@heap-s)
```

Comment:
```suggestion
- Enh #872: Use `#[\SensitiveParameter]` attribute to mark sensitive parameters (@heap-s)
```

Reply:
I believe that I've made all of your suggested changes, sorry this has been quite the process. Now that I understand the process for contributing a bit better here, I can make this easier in the future.

Reply:
It's OK. Thanks for contribution 👍